### ISSUE TITLE
WCAG 1.1.1: Images must have alternate text (#.logo-img-COjMQi)

### ISSUE THREAD

**Issue by zcjess** [zcjess/Problem-Roulette#6]
#### Issue
`Images must have alternate text` ([image-alt](https://accessibilityinsights.io/info-examples/web/image-alt))
#### Target application
[Problem Roulette | AES220](https://staging.problemroulette.ai.umich.edu/courses/AES220-23/)
#### Element path
#monica-content-root,.logo-img-COjMQi
#### Snippet
`<img src="chrome-extension://ofpnmcalabcbjgholdjcjblkibolbppb/static/monicaLogo.png" class="logo-img-COjMQi" style="width: 24px; height: 24px; border-radius: 4px;">`
#### Environment
Chrome version 114.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

![image](https://github.com/zcjess/Problem-Roulette/assets/90011962/3866a896-95ca-4cb6-b989-0ec801d33c21)

**Comment by zcjess** [comment]
no name, no role, not focusable
![image](https://github.com/zcjess/Problem-Roulette/assets/90011962/8fb8393a-d302-4dcc-82da-43e0687ed1b6)

**Comment by zcjess** [comment]
![image](https://github.com/zcjess/Problem-Roulette/assets/90011962/e417fc0c-e58b-41f0-a614-e8952d26272e)
https://staging.problemroulette.ai.umich.edu/AES220/SessionManager/156390/


Path | .member-avatar
